screens/GameScreen.java

```diff
@@ -172,7 +183,10 @@ public void paintComponent(Graphics g) {
 			target.draw(g);
 		}
 		player.draw(g);
-		for(DamageMarker marker : damageMarkers) {
+		for (Animation animation : animations) {
```

Comment:
if we try reducing the size of the arraylists anymore, problems will arise with rendering that will be more of a hassle to deal with than simply having a few arraylists